src/app/content/components/ContentLink.tsx

```diff
@@ -30,7 +30,7 @@ interface Props {
   navigate: typeof push;
   currentPath: string;
   hasUnsavedHighlight: boolean;
-  search: { query: string | null };
+  search: Record<string, string | null | undefined>;
```

Comment:
probably `null` is a mistake here, i don't see how it could be null right?`

Reply:
It is because we are doing:
```
search: ({
      ...
      'query': selectSearch.query(state),
      ...
    }),
```
and `selectSearch.query(state)` may return `null`

Reply:
maybe it should be changed to `undefined`?